com.github.leeroymanea.synology-note-station.yaml

```
@@ -0,0 +1,40 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: com.github.leeroymanea.synology-note-station
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+arch:
+  - x86_64
+tags:
+  - proprietary
+command: /app/opt/synology-note-station/launch.sh
```

```suggestion
command: /app/opt/synology-note-station/launch.sh
```

This should be installed or synlinked to `/app/bin`, then command should be `command: launch.sh`

---

done